week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,7 +38,8 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+arr.splice(index, 1);
+  return arr;
 }
```

Comment:
This is correct, for best practice store the parameter in a variable then apply the splice method to that variable.

Reply:
done